### ISSUE TITLE
Write more thank you messages for vending machines

### ISSUE THREAD

**Issue by Tayrtahn** [space-wizards/space-station-14#23803]
## Description

I implemented custom thank you messages for the different vending machines, but I'm not really a writer and we could use some more messages to add some variety. If anyone wants to do some writing, here's an opportunity to expand on the flavor of each vendor. Feel free to edit or remove the ones I wrote!

**Additional context**

The messages are localized, with the text in Fluent files in Resources/Locale/en-US/advertisements/vending/. The id strings then need to be added to the matching YAML files in Resources/Prototypes/Catalog/VendingMachines/Advertisements/.

**Comment by Tayrtahn** [comment]
I wouldn't call this completely done yet, though the PR above did help!

**Comment by metalgearsloth** [comment]
We have 100s I really don't think we need any more.

**Comment by Tayrtahn** [comment]
100s of advertising messages, yes. But the thank you messages when closing the UI are a bit repetitive still.

**Comment by UbaserB** [comment]
> 100s of advertising messages, yes. But the thank you messages when closing the UI are a bit repetitive still.

yeah, most vendors still speak the standard "thank you for using XX" And even if they do have a custom message, it is at most 3 which is fine but on the smaller side.
